sanic/sanic.py

```diff
@@ -47,9 +47,8 @@ def response(handler):
     # Decorator
     def exception(self, *exceptions):
         """
-        Decorates a function to be registered as a route
-        :param uri: path of the URL
-        :param methods: list or tuple of methods allowed
+        Decorates a function to be registered as a handler for exceptions
+        :param *exceptions: exceptions
```

Comment:
exception(variable in this method) is tuple, but this method (or *exception) does not receive tuple. So I think "exceptions" is better.